includes/DefaultSettings.php

```diff
@@ -5273,6 +5273,12 @@
 $wgAjaxLicensePreview = true;
 
 /**
+ * @name $wgEnableWikiaTOC
+ * enable Wikia JS driven TOC
+ */
+$wgEnableWikiaTOC = true;
+
+/**
```

Comment:
Why not move the include of the WikiaTOC extension here, if it's enabled sitewide?